### ISSUE TITLE
Pixmob Bracelet Wave Radio Frequency Hack.

### ISSUE THREAD

**Issue by ericmargay** [danielweidman/pixmob-ir-reverse-engineering#24]
In the recent concerts they are using this kind of bracelet. I am searching to the web if there is any hack. Apparently doesn't exist info about it. Do you think IR bracelet has the same chip as the RF bracelet ?.

Adding some pictures.
![IMG_5734](https://user-images.githubusercontent.com/48745498/232956769-e07efa1a-e6e6-4426-b7af-f39a1e3aa138.jpg)
![IMG_5735](https://user-images.githubusercontent.com/48745498/232956772-eae482b4-8460-4e70-9916-519cd1d2bbd9.jpg)
![IMG_5736](https://user-images.githubusercontent.com/48745498/232956776-535318f4-e5a3-4f2f-b558-14230a60f8c9.jpg)

**Comment by sean1983** [comment]
Hi, Thanks for your post, We are in the process of trying to decode them,  we had only had them a few days,   I have done my research on the main MCU IC and it looks to be the same,  Also the new addition Radio / OOK Decoder IC is interesting and the datasheet states it to be a direct replacement for Infrared,  So I am betting that IC sends the same old codes to the MCU after being demodulated/decoded from the OOK Modulation that is used via RF.  

But if you have a Flipper or HackRF, or DIY 915mhz  ardunio TX, Then we already have a few codes that will work.

**Comment by ericmargay** [comment]
> @ericmargay What concert did you get this from, by the way? I can share recordings of the signals soon

In Mexico City a mexican musical named Grupo Firme, in Foro Sol, February this year.

**Comment by ericmargay** [comment]
> @ericmargay What concert did you get this from, by the way? I can share recordings of the signals soon

I can share also in the next concerts.

**Comment by ericmargay** [comment]
> Hi, Thanks for your post, We are in the process of trying to decode them, we had only had them a few days, I have done my research on the main MCU IC and it looks to be the same, Also the new addition Radio / OOK Decoder IC is interesting and the datasheet states it to be a direct replacement for Infrared, So I am betting that IC sends the same old codes to the MCU after being demodulated/decoded from the OOK Modulation that is used via RF.
> 
> But if you have a Flipper or HackRF, or DIY 915mhz ardunio TX, Then we already have a few codes that will work.

I will get a HackRF to use the codes. Contact me to follow-up.

**Comment by TieHaxJan** [comment]
Hey guys, 
from 25th of April to 28th of April was the ProLight&Sound Fair in Frankfurt, Germany, Pixmob was there and I've got 9 of the Radio Frequency Controlled Bracelets, if I can help in any way, just contact me, I also can get ahold of a HackRF from my University. Sadly I was not able to bring any device to record their signals and also on the fair there was way to much radio noise, I would assume.
Cheers.

**Comment by dontic** [comment]
> Hi, Thanks for your post, We are in the process of trying to decode them, we had only had them a few days, I have done my research on the main MCU IC and it looks to be the same, Also the new addition Radio / OOK Decoder IC is interesting and the datasheet states it to be a direct replacement for Infrared, So I am betting that IC sends the same old codes to the MCU after being demodulated/decoded from the OOK Modulation that is used via RF.
> 
> But if you have a Flipper or HackRF, or DIY 915mhz ardunio TX, Then we already have a few codes that will work.

@sean1983 I have one of these Bracelets from a Swedish House Mafia concert and a Flipper, sadly I did not have it then so I do not have any recordings. Let me know if I can help in any way.

**Comment by dontic** [comment]
@sean1983 yes, pity I did not tinker with it sooner. This concert was in Madrid, October 14 2022. The venue was a new big dedicated hall in IFEMA, Madrid's fair grounds. Photo attached.

In [PIXMOB's site](https://www.pixmob.com/products/detail/led-wristbands) if you head to the Waveband  section it actually states "As seen on Swedish House Mafia and more.".

Will gladly join the discord, see you all there.
![IMG_8068](https://user-images.githubusercontent.com/25914304/235969435-4efc1b79-a29c-401d-acfc-07e51acac7de.jpg)

**Comment by below** [comment]
![IMG_5785](https://github.com/danielweidman/pixmob-ir-reverse-engineering/assets/26421/9815e615-39e0-47da-962b-50c41cda12da)
![IMG_5784](https://github.com/danielweidman/pixmob-ir-reverse-engineering/assets/26421/2ddb90b1-51e3-41ee-9015-a820d72de924)

Is this the device we are talking about (The Weeknd: After Hours Till Dawn)? I have two of them, both not deactivated. My questions:

* Can they be opened (relatively) non-destructively?
* What do I need to make them blink?
* Can I help anyone by donating my second device?
